plansys2_executor/test/test_behavior_trees/plansys2_action_bt.xml

```diff
@@ -0,0 +1,11 @@
+<Sequence name="ACTION_ID">
+WAIT_AT_START_ACTIONS
+  <ApplyAtStartEffect action="ACTION_ID"/>
+  <ReactiveSequence name="ACTION_ID">
+    <CheckTimeout action="ACTION_ID"/>
```

Comment:
I would leave the default BT without the `CheckTimeout` node because it is particular to an application that needs it. In case of needing it, you can pass your custom action BT with the `default_action_bt_xml_filename` parameter.

Reply:
I haven't actually changed the default BT. I just created a test_behavior_tree folder in the test directory and added this BT to test the action timeout capability. See action_timeout test in executor_test.cpp. To make this a bit clearer, however, I changed the name of this file from plansys2_action_bt.xml to test_action_timeout_bt.xml.